docs/get_started.md

```diff
@@ -1,6 +1,9 @@
-# Getting Started
+<img width="200" alt="Testrun logo" src="https://user-images.githubusercontent.com/7399056/221927867-4190a4e8-a571-4e40-9c2b-65780ad9264c.png" alt="Testrun">
 
-It is recommended that you run Testrun on a standalone machine running a fresh-install of Ubuntu 22.04.3 LTS.
+
+## Getting Started
+
+It is recommended that you run Testrun on a standalone machine running a fresh install of Ubuntu 22.04.4 or 24.04 LTS.
```

Comment:
Should match main readme versions

`PC running Ubuntu LTS 20.04, 22.04 or 24.04 (laptop or desktop)`